readme.adoc

```diff
@@ -884,6 +884,7 @@ jbang edit --open=[editor] helloworld.java
 ----
 
 The editor used will be what is specified as the argument to `--open` or default to `$JBANG_EDITOR`, `$VISUAL` or `$EDITOR` in that order.
+The editor command must be available on the PATH to be executed from jbang. If you are executing `jbang edit --open=code helloworld.java` a `code` executable (visual studio code) must be on the PATH.
```

Comment:
You can actually also provide the full path to the executable, eg: `--open=/usr/bin/code`